bin/crawl

```diff
@@ -21,7 +21,12 @@ export S3_BUCKET=`echo ${VCAP_SERVICES} | jq -r ".s3[0].credentials.bucket"`
 # Put Scrapy metadata files in a timestamped directory.
 DIR_NAME=`TZ=UTC date`
 
-scrapy crawl fac -a cfda=11 -t csv -o "s3://${S3_BUCKET}/fac-crawls/${DIR_NAME}/11.csv"
-scrapy crawl fac -a cfda=20 -t csv -o "s3://${S3_BUCKET}/fac-crawls/${DIR_NAME}/20.csv"
-scrapy crawl fac -a cfda=47 -t csv -o "s3://${S3_BUCKET}/fac-crawls/${DIR_NAME}/47.csv"
+# Crawl documents for each CFDA agency prefix.
+# NOTE: Currently, this will limit the crawl to all 2019 documents.
+for AGENCY_PREFIX in '01' '03' '04' '05' '06' '07' '08' '09' '10' '11' '12' '13' '14' '15' '16' '17' '18' '19' '20' '21' '22' '23' '27' '29' '30' '32' '33' '34' '36' '39' '40' '42' '43' '44' '45' '46' '47' '53' '57' '58' '59' '60' '61' '62' '64' '66' '68' '70' '77' '78' '81' '82' '83' '84' '85' '86' '87' '88' '89' '90' '91' '92' '93' '94' '95' '96' '97' '98' '99'
```

Comment:
Wondering if we can we do `seq -w 2 99` instead and let the crawler terminate if there isn't an actual corresponding agency prefix (e.g., `02`)?

Reply:
Yeah that is cleaner, but what I'm finding with trying to run this is that we're getting out of memory errors on some of them, in the sandbox. For instance ATM I've had to split `10` into `10.1`, `10.2`, etc, or order to avoid the errors.

This was supposed to be a quick and dirty way to quickly get everything crawled, but it's ending up being a hassle.. if you have any thoughts they'd be welcome.

Reply:
Ugh, yea, no great solution comes to mind. Do we know the distribution of each agency prefix -- it seems like they're not all created equally (some have more documents than others)?

Reply:
@cantsin I ran into another out of memory error last night, so to avoid more hassle on this, I'm running a crawl now with the `seq` approach, but segmenting by tenths (`XX.Y` prefix).

Reply:
Great, works for me.